backend/service/src/universal_codes/entity_filter.rs

```diff
@@ -21,7 +21,7 @@ pub fn dgraph_filter_from_v1_filter(filter: EntitySearchFilter) -> DgraphFilter
     // TODO: Better Search!
 
     // mSupply has a `special` way of querying that v1 API uses.
-    // and we need to set the type to ["Unit", "DoseStrength"] (in clause, where as normally this is a single string)
+    // and we need to set the type to ["Unit", "DoseStrength", "Presentation", "ExtraDescription"] (in clause, where as normally this is a single string)
```

Comment:
BTW: I ran into this `single string` problem when trying to update mSupply.
I think we should probably make type accept an array unless that somehow breaks mSupply?

Reply:
It is quite nice to be able to tweak the mSupply behaviour for the server, but it's a hacky method!